draft-ppm-protocol.md

```diff
@@ -1257,6 +1349,12 @@ but server implementations may also opt out of participating in a PPM task if
 the minimum batch size is too small. This document does not specify how to
 choose minimum batch sizes.
 
+The PPM parameters also specify the maximum number of times a report can be
+used. Some protocols, such as Hits, require reports to be used in multiple
+batches spanning multiple collect requests. At the same time, using a report in
```

Comment:
Yup, good call. Maybe this distinction would be more clear if we renamed "max_report_lifetime" to "max_batch_lifetime", since what we're really talking about is how many times a batch of reports can be queried.